web/assets/javascripts/metrics.js

```diff
@@ -0,0 +1,41 @@
+function init() {
+  const ctx = document.getElementById("metrics-chart");
+  const colors = JSON.parse(ctx.getAttribute("data-colors"));
+  const labels = JSON.parse(ctx.getAttribute("data-labels"));
+  const datasets = Object.entries(
+    JSON.parse(ctx.getAttribute("data-series"))
+  ).map(([k, v], i) => {
+    return {
+      label: k,
+      data: v,
+      borderColor: colors[i],
+      backgroundColor: colors[i],
+      borderWidth: 2,
+      pointRadius: 2,
+    };
+  });
+  const options = {
+    aspectRatio: 4,
+    scales: {
+      y: {
+        beginAtZero: true,
+      },
+    },
+    interaction: {
+      mode: "x",
+    },
+    plugins: {
+      legend: {
+        display: false,
```

Comment:
I opted the hide the default legend because I thought it was ugly. I think the swatches in the data table are just as effective and look a lot nicer.

![CleanShot 2022-08-10 at 16 49 21@2x](https://user-images.githubusercontent.com/372/184017482-99ce6444-fcb6-420f-8a94-3d471895ce69.png)

We could also potentially make the data table swatches interactive if, for example, we only wanted to initially show the top 5 job classes in the chart. You could use the swatch to toggle a job class' visibility in the chart. 🤷‍♂️